docker/tango/ska-python-buildenv/Dockerfile

```diff
@@ -70,6 +70,7 @@ COPY --chown=tango:tango Pipfile.lock Pipfile.lock
 RUN if [ -f "/usr/lib/x86_64-linux-gnu/libboost_python3-py37.so" ] && [ ! -f "/usr/lib/x86_64-linux-gnu/libboost_python-py37.so" ]; then \
     ln -s /usr/lib/x86_64-linux-gnu/libboost_python3-py37.so /usr/lib/x86_64-linux-gnu/libboost_python-py37.so; \
     fi; \
+    pipenv run pip install numpy==1.17.2; \
```

Comment:
The 'pipenv run' statement enforces install order: install numpy first (stage 1), then everything else in the pipfile (stage 2). The version in 'pipenv run' statement should match the version in the pipfile, otherwise a different version of numpy might be installed in stage 2 and there'll be confusion as to which version PyTango is built against.